src/events/ready.ts

```diff
@@ -1,10 +1,13 @@
-import { Events, Client } from "discord.js";
+import { Events, Client, TextChannel } from "discord.js";
 import cron from "node-cron";
 import dotenv from "dotenv";
 import { saveAuditLogs } from "@/commands/save-audit-logs.js";
 import { updateMessage } from "@/countdown-manager.js";
 import { updateSubsectionRoles } from "@/events/member-update.js";
 import fs from "node:fs";
+import persist from "node-persist";
```

Comment:
If you change the esp32 to give a heartbeat instead of when changes occur, you no longer need to use node-persist.

Reply:
node-persist is used to ensure the bot remembers the message ID of the embed message, so as not to create duplicate messages (this is so it can remove the old message and post a new one when a different state is detected)

Reply:
If not saved in the ./resources folder, it won't be persisted across restarts or updates as docker always will start with a clean filesystem, so just watch for that footgun.

Reply:
TODO: only update the message on a new state (not every state)